### ISSUE TITLE
[UX] File system settings: keep the "Default download method" field close to the public/private path fields.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1836]
...so it can be along with the rest of the path fields. It seems a bit "disconnected" the way it is currently:

![backdrop-file_system_settings-move_ui_translation_dir_field_above_transliteration_settings_section](https://cloud.githubusercontent.com/assets/2423362/15092923/55b79ac8-14bb-11e6-93cb-437e01225ae9.png)

**Comment by klonos** [comment]
Actually, there's something really weird happening when the Locale and Language modules are enabled. It seems that the "Default download method" section "gains weight" and moves down the form (or the "Transliteration" section moves up, depending on how one sees this)...

...before enabling the modules (the "Transliteration" section comes last):

![backdrop-file_system_settings-before_enabling_locale_and_language-transliteration_section_comes_last](https://cloud.githubusercontent.com/assets/2423362/15092994/42d7f162-14bd-11e6-97f2-508379587760.png)

...and after enabling the modules (the "Transliteration" section bubbles up):

![backdrop-file_system_settings-after_enabling_locale_and_language-transliteration_section_moves_up](https://cloud.githubusercontent.com/assets/2423362/15093003/6ac57bf4-14bd-11e6-9254-b5c82c578682.png)

**Comment by klonos** [comment]
...so if we make sure that the "Transliteration" section remains as the last section on that form, then I guess the "Interface translation directory" field would come after the rest of the directory/path settings.

**Comment by dboulet** [comment]
There is another problem, in that the Interface translations directory input field doesn’t actually work. Another solution might be to simply remove it.

See: https://github.com/backdrop/backdrop-issues/issues/1447

**Comment by klonos** [comment]
Yeah, as you can see form my last comments, the point is to actually keep the "Transliteration" section last so that the "Default download method" section is not "disconnected" from the public and private path fields that are placed at the top of that form. So, even if we end up completely removing the translation dir field, this issue here is still valid.

**Comment by klonos** [comment]
...better issue title to communicate that.
